### PR TITLE
Fix JDBC TTL to delete additional tables data.

### PR DESCRIPTION
**Pull request by wankai123**
Previous version the JDBC additional table missed  column `timeBucket`, and the TTL timer couldn't delete data from those 
tables.

- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [X] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

**Review by wu-sheng — DISMISSED** [review]

**Review by wu-sheng — DISMISSED** [review]

**Comment by kezhenxu94** [comment]
We need to pin the e2e to a fixed version https://github.com/apache/skywalking/blob/77ba2d553d77a4acacd8b2498cb09ea983a9baf4/.github/workflows/skywalking.yaml#L567

File: .github/workflows/skywalking.yaml [hunk 1]
```diff
@@ -564,7 +564,7 @@ jobs:
         run: |
           echo "${{ matrix.test.env }}"  >> $GITHUB_ENV
       - name: ${{ matrix.test.name }}
-        uses: apache/skywalking-infra-e2e@main
+        uses: apache/skywalking-infra-e2e@d71bbba95f95d6629db199e4bc0c0e097ae4f2dc
```
Comment (wu-sheng) [inline .github/workflows/skywalking.yaml:567]: FYI @kezhenxu94 Feel free to update the test policy after this PR is merged.

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/SQLDatabase.java [hunk 2]
```diff
@@ -113,4 +113,26 @@
...
+    @Target({ElementType.TYPE})
+    @Retention(RetentionPolicy.RUNTIME)
+    @Repeatable(MultipleExtraColumn4AdditionalEntity.class)
+    @interface ExtraColumn4AdditionalEntity {
+        String additionalTable();
+        String parentColumn();
+    }
+
+    /**
+     * The support of the multiple {@link ExtraColumn4AdditionalEntity}s on one field.
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/SQLDatabase.java:131]:
```suggestion
     * The support of the multiple {@link ExtraColumn4AdditionalEntity}s on the class.
```

File: oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/SQLDatabase.java [hunk 3]
```diff
@@ -113,4 +113,26 @@
         String[] additionalTables();
         boolean reserveOriginalColumns() default false;
     }
+
+    /**
+     * Support add an extra column from parent to an additional table.
+     * This column will be created in both the primary and additional tables.
+     * Notice: This annotation should be declared on the subclasses.
```
Comment (wu-sheng) [inline oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/annotation/SQLDatabase.java:120]:
```suggestion
     * Support add an extra column from the parent classes as a column of the additional table.
     * This column would be created in both the primary and additional tables.
     * Notice: This annotation should be declared on the leaf subclasses.
```

**Review by wu-sheng — DISMISSED** [review]

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**